website/src/components/layout.css

```diff
@@ -251,7 +251,7 @@ h2 {
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```

Comment:
This file does not exist anymore. Please use `components/misc/global-style` instead.